### ISSUE TITLE
"keybase pull" should update GPG keyring for tracked users

### ISSUE THREAD

**Issue by briantopping** [keybase/keybase-issues#1329]
When I track users, my intuition is that ```keybase pull``` should bring their pubkeys into my keychain. Am I misunderstanding something important?

```
Brians-MBP-2:web3 topping$ keybase version
keybase (keybase.io CLI) v0.7.3
- node.js v0.10.35
- gpg (GnuPG/MacGPG2) 2.0.26
- libgcrypt 1.6.2
Identifies as: 'keybase.io node.js client v0.7.3 darwin'
```

**Comment by palmerit** [comment]
@briantopping I'd agree with this.  I use keybase across a few machines, and I always thought 'keybase pull' should do exactly that.  pull any tracked user into the public key ring.  or maybe a 'keybase sync' option.  As it exists right now, using keybase across more than 1 or 2 machines is.. tedious.

**Comment by Thracky** [comment]
I agree with both @briantopping and @palmerit here. Having to locally track everyone that is already listed as being tracked in the remote list is counter-intuitive.

**Comment by beatrohrer** [comment]
I second that.

As far as I know, when people change their keys, I don't get a notification and my pub keys don't get updated. People must tell me, that they've changed their key and I have to manually re-track them. This is tedious and defeats the whole point of keybase as a key management place.

**Comment by maxtaco** [comment]
I third, this feature makes a ton of sense. We are adding this feature this week to the new Go client (cc: @oconnnor663), but if we have a spare moment, we can back port this feature to the node client.

**Comment by ZacharyDuBois** [comment]
I am in here now too.

**Comment by glyph** [comment]
I just experienced some confusion as well when `keybase pull` did not do this.

**Comment by ZacharyDuBois** [comment]
@glyph For now, just go through your keybase profile and re-track everyone.
